sdk/appconfiguration/azure-data-appconfiguration/src/test/resources/session-records/addConfigurationSettingNullKey.json

```diff
@@ -1,10 +1,10 @@
 {
```

Comment:
Don't update *.json files if you aren't modifying the test case. This creates unnecessary diffs and additions to PRs.

Reply:
After chagne to PLAYBACK mode with unchanged json files, it almost failed all tests beside testAzureConfiguration and AadCredentialTest. The existing json files replacement will be needed in this case. So i will leave the PR unchanged.

Reply:
I tried to revert back to unchanged JSON files, but it failed almost all tests besides AadCredentialTest and testAzureConfiguration two tests. So I will still stay what it is in this case.